documentation/modules/exploit/linux/http/pandora_fms_events_exec.md

```diff
@@ -0,0 +1,110 @@
+## Vulnerable Application
+This module exploits a vulnerability (CVE-2020-13851) in Pandora FMS versions 7.0 NG 742, 7.0 NG 743, and 7.0 NG 744
+(and perhaps older versions) in order to execute arbitrary commands.
+
+The module first connects to the target's `index.php` page in order to verify the version number, which should be displayed here.
+If the version indicates the target is likely vulnerable, the module will try to authenticate using the credentials provided by the user.
+If no custom credentials are provided, the module will use the default Pandora FMS credentials, which are `admin:pandora`.
+
+After authenticating, the module attempts to exploit CVE-2020-13851, which is a command injection vulnerability
+in the `Events` feature of Pandora FMS. Specifically, this flaw allows users to execute arbitrary commands via
+the `target` parameter in HTTP POST requests to the `Events` function.
+In order to obtain remote code execution, the module will attempt to issue a malicious HTTP POST request to the `Events` function,
+with the `target` parameter set to contain the payload.
+
+If a shell is obtained, the module will try to obtain the local MySQL database password via a simple `grep` command on the plaintext
+`/var/www/html/pandora_console/include/config.php` file.
+The default MySQL administrative user is `root` and the default password for the official CentOS virtual appliance ISO is `pandora`.
+For the official Docker container, the default MySQL password is `avwwoyqk`. This password can subsequently be used
+in order to query the database and to escalate the privilege of any Pandora FMS account to an administrator.
+
+Valid credentials for a Pandora FMS account are required for the module to work. The account does not need to have admin privileges.
+This module has been successfully tested on Pandora 7.0 NG 744 running on CentOS 7 (the official virtual appliance ISO for this version).
```

Comment:
Might want to document setup in this section.

https://github.com/rapid7/metasploit-framework/pull/13730#issuecomment-644917728

Reply:
done!